package.json

```diff
@@ -23,26 +23,26 @@
     "@angular/platform-browser-dynamic": "20.1.4",
     "@angular/router": "20.1.4",
     "@fortawesome/fontawesome-free": "5.15.4",
-    "@ngx-translate/core": "16.0.4",
-    "@ngx-translate/http-loader": "16.0.1",
-    "ag-grid-angular": "33.3.0",
-    "ag-grid-community": "33.3.0",
+    "@ngx-translate/core": "17.0.0",
+    "@ngx-translate/http-loader": "17.0.0",
+    "@primeng/themes": "20.3.0",
+    "ag-grid-angular": "34.0.0",
+    "ag-grid-community": "34.0.0",
     "angular-split": "20.0.0",
     "bootstrap": "4.6.2",
     "core-js": "3.44.0",
-    "date-fns": "2.30.0",
+    "date-fns": "4.1.0",
     "interactjs": "1.10.27",
     "jquery": "3.7.1",
     "mobile-drag-drop": "3.0.0-rc.0",
     "nanobar": "0.4.2",
-    "pako": "1.0.11",
+    "pako": "2.1.0",
```

Comment:
This dependency is tricky. I would ask for someone using it to be sure we can update without regrets